src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
+      placement="bottom"
+      trigger="click"
+      level="modal"
+      @update:open="clickTrigger"
+    >
+      <div class="ec-filter-popover__filter-item">
+        <a class="ec-filter-popover__label">{{ label }}</a>
```

Comment:
Can we add a data-test for this anchor so we have something to target in integration tests.